### PR TITLE
Themes 1222 - Logical Properties on Article Body Block

### PR DESCRIPTION
**Pull request by blakeganderson**
## Description

This PR updates the logical properties on the article-body-block.

## Jira Ticket

- [THEMES-1222](https://arcpublishing.atlassian.net/browse/THEMES-1222)

## Acceptance Criteria

1. The above logical properties are applied and tested on the article-body-block.
2. Storybook is updated with RTL. 

## Test Steps

1. Checkout this branch `git checkout THEMES-1222`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block`
3. Go to http://localhost/pf/2022/09/15/the-kitchen-sink-of-composer-features/?_website=the-gazette
4. Verify it matches the designs in Figma.
5. Set `"textDirection": "rtl"` in the `"siteProperties"` object of the `blocks.json`.
6. Restart PageBuilder and verify that the styling adapts to the new text direction.

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-1222]: https://arcpublishing.atlassian.net/browse/THEMES-1222?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by malavikakoppula — CHANGES_REQUESTED** [review]
@blakeganderson article-body-block is also in commerce. we'll need logical properties to be applied there as well.

**Comment by blakeganderson** [comment]
@malavikakoppula good catch, I'll go back and make the commerce changes

**Comment by blakeganderson** [comment]
@malavikakoppula I have made the updates for commerce

File: blocks/article-body-block/themes/news.json [hunk 1]
```diff
@@ -301,7 +312,10 @@
 			"default": {
 				"box-shadow": "(inset 0 -5px 0 0 var(--border-color), inset 0 5px 0 0 var(--border-color))",
 				"gap": "var(--global-spacing-4)",
-				"padding": "var(--global-spacing-6) 0 var(--global-spacing-5)",
+				"padding-block-start": "var(--global-spacing-6) 0",
+				"padding-inline-end": "var(--global-spacing-5)",
+				"padding-block-end": "var(--global-spacing-6) 0",
+				"padding-inline-start": "var(--global-spacing-5)",
```
Comment (malavikakoppula) [inline blocks/article-body-block/themes/news.json]: @blakeganderson  Here's is the doc for padding: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

"padding-block-start": "var(--global-spacing-6) ",
"padding-inline-end": "0",
"padding-block-end": "var(--global-spacing-5)",
"padding-inline-start": "0",

**Review by malavikakoppula — CHANGES_REQUESTED** [review]

File: blocks/article-body-block/themes/news.json (hunk 1 above)
Comment (blakeganderson) [reply blocks/article-body-block/themes/news.json]: good catch, fixed!

**Review by malavikakoppula — APPROVED** [review]
LGTM

**Pull request merged**